plugins/lightspeed-backend/src/service/router.ts

```diff
@@ -24,25 +37,66 @@ export async function createRouter(
     response.json({ status: 'ok' });
   });
 
+  router.get(
+    '/conversations/:conversation_id',
+    validateLoadHistoryRequest,
+    async (request, response) => {
+      const conversation_id = request.params.conversation_id;
+      const historyLength = Number(request.query.historyLength);
+
+      const loadhistoryLength: number = historyLength || DEFAULT_HISTORY_LENGTH;
+      try {
+        const history = await loadHistory(conversation_id, loadhistoryLength);
+        response.status(200).json(history);
+        response.end();
+      } catch (error) {
+        const errormsg = `Error: ${error}`;
+        logger.error(errormsg);
+        response.status(500).json({ error: errormsg });
```

Comment:
no need to explicitly append`Error:` in front of the error (inside string literal) because it  gets appended twice.
```
   {  "error": "Error: Error: unkown conversation_id: user1+1q2w3e4r-qwer1234" }
```


```suggestion
        const errormsg = `${error}`;
        logger.error(errormsg);
        response.status(500).json({ error: errormsg });
```

Reply:
updated